app/models/miq_worker.rb

```diff
@@ -52,6 +53,14 @@ def self.workers
     workers_configured_count
   end
 
+  def self.scalable?
+    maximum_workers_count.kind_of?(Integer) && maximum_workers_count == 1
```

Comment:
Shouldn't this be the opposite logic? Something is scalable if the maximum is greater than 1, right?

Reply:
D'oh yeah changed the name but forgot to invert the logic, on that note does anyone know what other type maximum_workers_count could be?

Reply:
String? `"three"` :trollface: 

But in reality if it's not an integer `"three" == 1` is still `false` so the type check can probably be dropped.

Reply:
Haha yeah I just had to check real quick that we didn't have any lambdas or something there

Reply:
Done